### PR TITLE
User type in GET Requests

### PR DESCRIPTION
**Pull request by rajakvk**
Back end will return newly joined member when hitting /users?userType=new json end point.
We need to add new field called 'userType' in the db. 
Allowed userType values could be 'new', 'blocked', 'member'.
```js
  const allowedUserTypes = ['new', 'blocked', 'member']
```
Please review @ankurnarkhede

### CODE REVIEW

File: models/users.js [hunk 1]
```diff
@@ -54,11 +54,21 @@ const addOrUpdate = async (userData, userId = null) => {
...
  */
 const fetchUsers = async (query) => {
+  const allowedUserTypes = ['new', 'blocked', 'member']
+  let snapshot
   try {
-    const snapshot = await userModel
-      .limit(parseInt(query.size) || 100)
-      .offset((parseInt(query.size) || 100) * (parseInt(query.page) || 0))
-      .get()
+    if (typeof query.userType !== 'undefined' && allowedUserTypes.includes(query.userType)) {
```
Comment (sumitd94) [inline models/users.js:60]: is it needed to check for `undefined` ?
cant we use something like 
`if(query.userType && ....)`

**Review by sumitd94 — CHANGES_REQUESTED** [review]
One small change please.

File: models/users.js [hunk 2]
```diff
@@ -54,11 +54,21 @@ const addOrUpdate = async (userData, userId = null) => {
  * @return {Promise<userModel|Array>}
  */
 const fetchUsers = async (query) => {
+  const allowedUserTypes = ['new', 'blocked', 'member']
```
Comment (sumitd94) [inline models/users.js:57]: Please update the `swaggerDocs` and generate the API Schema accordingly by running the command
`npm run generate-api-schema`

**Review by sumitd94 — CHANGES_REQUESTED** [review]

File: models/users.js (hunk 2 above)
Comment (rajakvk) [reply models/users.js:57]: Updated swaggerDefinition.js file with additional field "userType".

**Comment by rajakvk** [comment]
@sumitd94 is it still relevant? If not, we may close

**Pull request closed without merging**